### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by r-jo**
### 3. No package modifications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +45,14 @@ There's a few complexities to take note of:
...
+## Possible Errors
+
+- Project won't start because of `Error occurred while proxying request localhost:8080/graphql ...`?
+    - Run `yarn workspace @app/db prisma generate`.
+    - Run `yarn g:dev-server` to check if the server starts up without any errors.
+    - Ensure you have [nvm](https://github.com/nvm-sh/nvm) installed and/or follow [this guide](https://stackoverflow.com/questions/11284634/upgrade-node-js-to-the-latest-version-on-mac-os) to upgrade your node to LTS (currently v14.x).
+    - Open a fresh terminal and run `yarn g:dev-project`.
+
+- `Subject/type may not be empty` error when trying to commit for the first time?
+    - Format your commit message as `addition(scope): what addition you made` where addition can be `fix`, `feature`, `refactor`, etc. `(scope)` can be `(global)`, `(frontend)`, `(backend)`, `(fullstack)`, `(docs)`. Your message should be a string of all lowercase letters that isn't sentence-case, start-case, pascal-case, or upper-case.
```
Comment (DevelopmentByDavid) [inline README.md:58]: Oh, right I forgot to go over this.  There's a prompt for this `yarn g:commit`. I'll accept this and add that too.

Comment (r-jo) [reply README.md:58]: Yeah I found the command in the package.json but I got an error saying the command does not exist.

Comment (DevelopmentByDavid) [reply README.md:58]: Oh, that's interesting.  Do you mind sending me the error you had?  I had intended for everyone to be able to use `yarn g:commit`.  So if it's not working I need to figure out why.

Comment (DevelopmentByDavid) [reply README.md:58]: I'm going to merge this PR, you can just send me it on slack.

**Pull request merged**